### PR TITLE
Feedback for article - VSI Flavors

### PR DESCRIPTION
**Pull request by caberos**
https://github.com/softlayer/githubio_source/issues/417

### CODE REVIEW

**Review by FernandoOjeda — APPROVED** [review]
Looks good for me.

File: content/article/vsiflavors.md [hunk 1]
```diff
@@ -101,3 +101,45 @@ slcli vs create --os=UBUNTU_LATEST_64 \
...
+
+# Getting a order of a VSI existent
+
+[SoftLayer_Virtual_Guest::getObject](https://softlayer.github.io/reference/services/SoftLayer_Virtual_Guest/getObject/) is where you can find out the order details of the VSI specific, use with a mask to getting the information
+
+In the response from getObject, will be a VSI information, and with information about their [order](https://sldn.softlayer.com/reference/datatypes/SoftLayer_Billing_Order/#items) with items.
+
+
+``` 
+https://api.softlayer.com/rest/v3.1/SoftLayer_Virtual_Guest/123456/getObject?objectMask=mask[billingItem[orderItem[order[items]]]]
```
Comment (ATGE) [inline content/article/vsiflavors.md:135]: tag the code block as bash and try to use curl format:
`curl -u $SL_USER:$SL_APIKEY -X GET -H "Accept: */*" -H "Accept-Encoding: gzip, deflate, compress"  'https://api.softlayer.com/rest/v3.1/SoftLayer_Virtual_Guest/112238162/getObject.json?objectMask=mask%5BbillingItem%5BorderItem%5Bpreset%5D%5D%5D'`

`slcli -vvv`  helps you to get the curl format:
`slcli --format=json -vvv call-api Virtual_Guest getObject --id=12345 --mask='mask[billingItem[orderItem[preset]]]'`

**Review by ATGE — CHANGES_REQUESTED** [review]
looks goods, just tag the code blocks as json/bash so they highlight properly

**Review by allmightyspiff — APPROVED** [review]

**Review by ATGE — APPROVED** [review]

File: content/article/vsiflavors.md [hunk 2]
```diff
@@ -24,7 +24,7 @@ Support for flavors has been added to the [SLCLI](https://github.com/softlayer/s
 In the response from getCreateObjectOptions, will be a key "flavors", with an array of different flavors and some information about their configuration.
 
 ```
```
Comment (ATGE) [inline content/article/vsiflavors.md:26]: tag as bash

**Review by ATGE — APPROVED** [review]

**Pull request merged**